mlibrary/pride
# Pride.Util.Paginater.getPossibleKeys and Pride.Util.Paginater.hasKey

# Overview
`Pride.Util.Paginater.getPossibleKeys()` was a function that returned an array. This has been converted to just an array. The tests for this property have been updated.

`Pride.Util.Paginater.hasKey()` was a one-line function that was only used in `Pride.Core.Query()`. The `hasKey()` code has been migrated over into `Query()` and the original files have been deleted.

## Anything else?
`Pride` was listed as in development for years at version `0.0.1`. With the new setup officially in place, `version` has been updated to `1.0.0`.

## Testing
1. Build the repository (`npm run build`).
2. Run the tests to make sure they pass (`npm run test`).
   * Break the new/updated unit tests to make sure they're working properly.
3. Navigate to your local [Search repository](https://github.com/mlibrary/search) and apply the newly generated files:
   1. Open `./package.json` and add `#Pride-Util-Paginater-getPossibleKeys` at the end of the `pride` dependency URL:
      ```bash
      "pride": "git+https://github.com/mlibrary/pride.git#Pride-Util-Paginater-getPossibleKeys"
      ``` 
   2. Do a clean install of Search:
      ```bash
      rm -rf node_modules && rm package-lock.json && npm install
      ``` 
   3. Start Search (`npm start`) and look around [the site](http://localhost:3000/everything). Check to see if there are any console errors, and everything still works as expected.
- Make sure the PR is consistent in these browsers:
